pybamm/solvers/scikits_dae_solver.py

```diff
@@ -69,6 +72,23 @@ def rootfn(t, y, ydot, return_root):
             return_root[:] = [event(t, y) for event in events]
 
         extra_options = {"old_api": False, "rtol": self.tol, "atol": self.tol}
+
+        # If no Jacobian provided (default), use autograd to compute the
+        # Jacobian. If autograd not installed, the solver will approximate the
+        # Jacobian (see SUNDIALS documentation).
```

Comment:
if you put this in the docstring you can put in a hyperlink to SUNDIALS docs (see [here](http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets))